css-layout-classes/styles.css

```diff
@@ -0,0 +1,30 @@
+.container {
+  display: flex;
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
```

Comment:
Your `.container` shouldn't have any properties related to `flex` on here. This won't necessarily always be the case, but just for this specific exercise.